healsparse/healSparseMap.py

```diff
@@ -1120,8 +1120,8 @@ def generate_healpix_map(self, nside=None, reduction='mean', key=None, nest=True
             sparsemap nside_sparse
         reduction : `str`
             If a change in resolution is requested, this controls the method to
-            reduce the map computing the mean, median, std, max or min of the
-            neighboring pixels to compute the "degraded" map.
+            reduce the map computing the mean, median, std, max, min, sum or
+            product of the neighboring pixels to compute the "degraded" map.
```

Comment:
I think that we should put quotes around the reduction options.  At the same time it is `"prod"` not `"product"`.  Maybe should be `computing the "mean", "median", "std" (standard deviation), "max", "min", "sum", or "prod" (product) of `...